tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/process_managers/default_processes/default_freespace_processes.h

```diff
@@ -49,28 +50,33 @@ TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
 namespace tesseract_planning
 {
-inline std::vector<ProcessGenerator::Ptr> defaultFreespaceProcesses()
+inline SequentialProcesses defaultFreespaceProcesses()
 {
+  SequentialProcesses sp;
+
   // Setup Interpolator
   auto interpolator = std::make_shared<SimpleMotionPlanner>("INTERPOLATOR");
   interpolator->plan_profiles["FREESPACE"] = std::make_shared<SimplePlannerDefaultPlanProfile>();
   auto interpolator_generator = std::make_shared<MotionPlannerProcessGenerator>(interpolator);
-  interpolator_generator->validators.emplace_back(&randomValidator);
+  sp.emplace_back(interpolator_generator, SequentialTaskType::CONDITIONAL_EXIT_ON_FAILURE);
```

Comment:
So this is a bit backwards from what I was intending (though not necessarily incorrect). I was intending it to try one until it succeeded. Now if I understand correctly it will always run all 3 (in a successful case).

How would we handle a case like
- Always run initialize seed (which should succeed)
- Run trajopt. Return if succeeds
- Run OMPL.
- Run OMPL with different parameters
- etc

Reply:
You  are correct, but you can easily create what you intended by switching out the task type for trajopt and ompl to CONDITIONAL_EXIT_ON_SUCCESS.

Reply:
@mpowelson Does the name SequentialFailureTreeTaskFlow still fit? Should it be renamed to something more general like MotionPlanningTaskFlow?

Reply:
Maybe just SequentialTaskFlow. You could add tasks that aren't motion planning specific. But it also isn't strictly a cascading error tree anymore.

Reply:
Sound good, SequentialTaskFlow it is.